transport_plugins/bled112/iotile_transport_bled112/bled112.py

```diff
@@ -537,15 +551,19 @@ def _process_scan_event(self, response):
         else:
             data = bytearray([])
 
+        self._scan_event_count += 1
+
         # If this is an advertisement packet, see if its an IOTile device
         # packet_type = 4 is scan_response, 0, 2 and 6 are advertisements
         if packet_type in (0, 2, 6):
             if len(data) != 31:
                 return
 
             if data[22] == 0xFF and data[23] == 0xC0 and data[24] == 0x3:
+                self._v1_scan_count += 1
                 self._parse_v1_advertisement(rssi, string_address, data)
             elif data[3] == 27 and data[4] == 0x16 and data[5] == 0xdd and data[6] == 0xfd:
+                self._v2_scan_count += 1
```

Comment:
I would add a one more statistic here that will be good to keep track of:

1. On v1 packets there is both an advertisement and a scan_response (when active scanning).  Having a separate counter for v1 advertisements and v1 scan_responses will let us know how many times we get an advertisement but miss its scan response.  The second counter can go before the call to  `_parse_v1_scan_response`.